includes/Classifai/Features/Feature.php

```diff
@@ -956,6 +956,10 @@ public function has_access(): bool {
 			$access             = ( ! in_array( static::ID, $opted_out_features, true ) );
 		}
 
+		if ( defined( 'WP_CLI' ) && WP_CLI ) {
+			$access = true;
```

Comment:
@dkotter can you verify this. I've added it so that a feature can run via WP_CLI.
cc: @iamdharmesh